core/services/fluxmonitor/flux_monitor_test.go

```diff
@@ -196,6 +196,7 @@ func TestPollingDeviationChecker_PollIfEligible(t *testing.T) {
 			fetcher := new(mocks.Fetcher)
 			fluxAggregator := new(mocks.FluxAggregator)
 			logBroadcaster := new(logmocks.Broadcaster)
+			logBroadcaster.On("IsConnected").Return(test.connected).Maybe()
```

Comment:
hmm do you mean, this can sometimes fail, or is not strict enough?
It's still deterministic, right? Just a bit permissive, but yeah i'll see what can be done